src/3d/qgs3dsceneexporter.cpp

```diff
@@ -242,17 +245,15 @@ void Qgs3DSceneExporter::processEntityMaterial( Qt3DCore::QEntity *entity, Qgs3D
     const Qt3DRender::QTexture2D *diffuseTexture = diffuseMapMaterial->diffuse().value< Qt3DRender::QTexture2D * >();
     if ( diffuseTexture != nullptr )
     {
-      const QVector<Qt3DRender::QAbstractTextureImage *> textureImages = diffuseTexture->textureImages();
-      QgsImageTexture *imageTexture = nullptr;
-      for ( Qt3DRender::QAbstractTextureImage *tex : textureImages )
+      for ( const Qt3DRender::QAbstractTextureImage *tex : diffuseTexture->textureImages() )
```

Comment:
can you leave the previous approach of storing the images in a const list first? Otherwise we'll get a detachment of the qt container and consequent clazy warnings.